### PR TITLE
docs: update general graphcast docs, add a page for 3la radio

### PR DESCRIPTION
**Pull request by hopeyen**
Update sections
- Design principle
- Graphcast intro
- SDK intro - a few subsections added
- POI new user configs
- 3LA page under the radio category

### CODE REVIEW

File: docs/graphcast/radios/3la-monitor.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+| `PERSISTENCE_FILE_PATH`    | Relative path. If set, the Radio will periodically store states of the program to the file in json format (off by default).                                                                                                        |
+| `DISCV5_ENRS`              | Comma separated ENRs for Waku Discv5 bootstrapping. Defaults to empty list.                                                                                                                                                        |
+| `DISCV5_PORT`              | Discoverable UDP port. Default: `9000`                                                                                                                                                                                             |
+| `ID_VALIDATION`            | Defines the level of validation for message signers used during radio operation. Options include: `no-check`, `valid-address`, `graphcast-registered`, `graph-network-account`, `registered-indexer`, `indexer` |
```
Comment (axiomatic-aardvark) [inline docs/graphcast/radios/3la-monitor.md:82]:
```suggestion
| `ID_VALIDATION`            | Defines the level of validation for message signers used during Radio operation. Options include: `no-check`, `valid-address`, `graphcast-registered`, `graph-network-account`, `registered-indexer`, `indexer` |
```

File: docs/graphcast/radios/3la-monitor.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+`COVERAGE` is used to specify the topic coverage level. It controls the range of topics (subgraph ipfs hashes) the Indexer subscribes to in order to process data and participate in the network.
+
+There are three coverage levels available:
+
+- **comprehensive**: Subscribe to on-chain topics, user-defined static topics, and subgraph deployments syncing on graph node. This level is useful for Indexers who want to compare NPOIs for all deployments syncing on their graph node even if they don't have an active allocations open (their stake will not be taken into account in attestation).
+- **on-chain**: Subscribe to on-chain topics and user-defined static topics. This is the default coverage level and is suitable for indexers who only want to compare data for deployments with active allocations.
+- **minimal**: Only subscribe to user-defined static topics. This level is for Indexers who want to limit their participation to specific topics of interest.
+
+#### Identity validaiton
+`ID_VALIDATION` is used to define level of validation for message signers used during radio operation.
```
Comment (axiomatic-aardvark) [inline docs/graphcast/radios/3la-monitor.md:97]:
```suggestion
`ID_VALIDATION` is used to define level of validation for message signers used during Radio operation.
```

File: docs/graphcast/radios/3la-monitor.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+Available Options:
+- **no-check**: No check on the message signature and do not verify the signer.
```
Comment (axiomatic-aardvark) [inline docs/graphcast/radios/3la-monitor.md:100]:
```suggestion
- **no-check**:  Doesn't perform check on the message signature and does not verify the signer.
```

File: docs/graphcast/radios/3la-monitor.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+- **valid-address**: signer needs to be a valid Eth address.
```
Comment (axiomatic-aardvark) [inline docs/graphcast/radios/3la-monitor.md:101]:
```suggestion
- **valid-address**: Requires the signer to be a valid Ethereum address.
```

File: docs/graphcast/radios/3la-monitor.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+- **graphcast-registered**: signer must be registered at Graphcast Registry.
```
Comment (axiomatic-aardvark) [inline docs/graphcast/radios/3la-monitor.md:102]:
```suggestion
- **graphcast-registered**: Requires the signer to be registered on the Graphcast Registry.
```

File: docs/graphcast/radios/3la-monitor.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+- **graph-network-account**: signer must be a Graph account.
```
Comment (axiomatic-aardvark) [inline docs/graphcast/radios/3la-monitor.md:103]:
```suggestion
- **graph-network-account**: Signer must be a Graph account.
```

File: docs/graphcast/radios/3la-monitor.md [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+- **registered-indexer**: signer must be registered at Graphcast Registry and correspond to an Indexer satisfying the indexer minimum stake requirement.
```
Comment (axiomatic-aardvark) [inline docs/graphcast/radios/3la-monitor.md:104]:
```suggestion
- **registered-indexer**:  Requires the signer to be registered on the Graphcast Registry and corresponds to an Indexer that satisfies the minimum stake requirement.
```

File: docs/graphcast/radios/3la-monitor.md [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+- **indexer**: signer must be registered at Graphcast Registry or is a Graph Account, and correspond to an Indexer satisfying the indexer minimum stake requirement.
```
Comment (axiomatic-aardvark) [inline docs/graphcast/radios/3la-monitor.md:105]:
```suggestion
- **indexer**: Requires the signer to be either registered on the Graphcast Registry or to be a Graph Account, and corresponds to an Indexer satisfying the minimum stake requirement.
```

File: docs/graphcast/radios/poi-radio.md [hunk 9]
```diff
@@ -116,7 +117,12 @@ See [Basic Configuration](#basic-configuration) above. The following environment
 | `PERSISTENCE_FILE_PATH`    | Relative path. If set, the Radio will periodically store states of the program to the file in json format (off by default).                                                                                                        |
 | `DISCV5_ENRS`              | Comma separated ENRs for Waku Discv5 bootstrapping. Defaults to empty list.                                                                                                                                                        |
 | `DISCV5_PORT`              | Discoverable UDP port. Default: `9000`                                                                                                                                                                                             |
+| `ID_VALIDATION`            | Defines the level of validation for message signers used during radio operation. Options include: `no-check`, `valid-address`, `graphcast-registered`, `graph-network-account`, `registered-indexer`, `indexer`. Default: `registered-indexer` |
```
Comment (axiomatic-aardvark) [inline docs/graphcast/radios/poi-radio.md:120]:
```suggestion
| `ID_VALIDATION`            | Defines the level of validation for message signers used during Radio operation. Options include: `no-check`, `valid-address`, `graphcast-registered`, `graph-network-account`, `registered-indexer`, `indexer`. Default: `registered-indexer` |
```

File: docs/graphcast/radios/poi-radio.md [hunk 10]
```diff
@@ -116,7 +117,12 @@ See [Basic Configuration](#basic-configuration) above. The following environment
...
 
+### Configurations explained
```
Comment (axiomatic-aardvark) [inline docs/graphcast/radios/poi-radio.md:122]:
```suggestion
### Configuration explained
```

File: docs/graphcast/radios/poi-radio.md [hunk 11]
```diff
@@ -125,6 +131,19 @@ There are three coverage levels available:
 - **on-chain**: Subscribe to on-chain topics and user-defined static topics. This is the default coverage level and is suitable for indexers who only want to compare data for deployments with active allocations.
 - **minimal**: Only subscribe to user-defined static topics. This level is for Indexers who want to limit their participation to specific topics of interest.
 
+#### Identity validaiton
+`ID_VALIDATION` is used to define level of validation for message signers used during radio operation. We recommend `registered-indexer` for most strict identity validation, while `indexer` and is a viable option for those who want to use the network before considering Grapchast ID registration. Choose sender identity validation for your radio base on your use case and security preference.
```
Comment (axiomatic-aardvark) [inline docs/graphcast/radios/poi-radio.md:135]:
```suggestion
`ID_VALIDATION` is used to define level of validation for message signers used during Radio operation. We recommend `registered-indexer` for most strict identity validation, while `indexer` is a viable option for those who want to use the network before considering Grapchast ID registration. You can choose a sender identity validation mechanism for your Radio, based on your use case and security preferences.
```

File: docs/graphcast/radios/poi-radio.md [hunk 12]
```diff
@@ -125,6 +131,19 @@ There are three coverage levels available:
...
+
+Available Options:
+- **no-check**: No check on the message signature and do not verify the signer of messages. All messages should pass the sender check.
+- **valid-address**: signer needs to be a valid Eth address. Messages should be traceable to an Ethers wallet.
+- **graphcast-registered**: signer must be registered at Graphcast Registry.
+- **graph-network-account**: signer must be a Graph account.
+- **registered-indexer**: signer must be registered at Graphcast Registry and correspond to an Indexer satisfying the indexer minimum stake requirement.
+- **indexer**: signer must be registered at Graphcast Registry or is a Graph Account, and correspond to an Indexer satisfying the indexer minimum stake requirement.
```
Comment (axiomatic-aardvark) [inline docs/graphcast/radios/poi-radio.md:143]: same suggestions as above

File: docs/graphcast/sdk/intro.md [hunk 13]
```diff
@@ -4,7 +4,31 @@ sidebar_position: 1
...
+
+1. **Connection to the Graphcast network**: Forms a communication network and provides an interface to subscribe to receive messages on specific topics and to broadcast messages onto the network. Allows for real-time communication between different nodes in the network.
+
+2. **Interactions with Graph entities**: This allows for necessary interactions with Graph node, Graph network subgraph, Graphcast registry.
+
+An example of a ping-pong Radio is provided in the examples folder, which leverages the base layer and defines the specific logic around constructing and sending messages, as well as receiving and handling them. This example can serve as a starting point for developers looking to build their own Radios.
+
+## Network Configurations
+
+A Graphcast radio can interact with many parts of The Graph network modularly. The network configurations actively supported by the team include `mainnet` (Ethereum mainnet and Arbitrum One) and `testnet` (Goerli and Arbitrum Goerli). You are free to define and use your own Graphcast Network and Graphcast Registry. This flexibility allows for a wide range of applications and use cases.
```
Comment (axiomatic-aardvark) [inline docs/graphcast/sdk/intro.md:21]:
```suggestion
A Graphcast Radio can interact with many parts of The Graph network modularly. The network configurations actively supported by the team include `mainnet` (Ethereum mainnet and Arbitrum One) and `testnet` (Goerli and Arbitrum Goerli). You are free to define and use your own Graphcast Network and Graphcast Registry. This flexibility allows for a wide range of applications and use cases.
```

File: docs/graphcast/sdk/intro.md [hunk 14]
```diff
@@ -4,7 +4,31 @@ sidebar_position: 1
...
+
+## Contributing
+
+Contributions are welcome and appreciated! Please refer to the Contributor Guide, Code Of Conduct, and Security Notes for this repository. These documents provide guidelines for how to contribute to the project in a way that is beneficial to all parties involved.
+
+## Upgrading and Testing
+
+Updates to the SDK will be merged into the main branch once their release PR has been approved. For testing, it is recommended to use `nextest` as your test runner. You can run the suite using the command `cargo nextest run`. Regular testing is crucial to ensure the stability and reliability of the software.
```
Comment (axiomatic-aardvark) [inline docs/graphcast/sdk/intro.md:29]:
```suggestion
Updates to the 3LA repository will be merged into the main branch once their release PR has been approved. For testing, it is recommended to use `nextest` as your test runner. You can run the suite using the command `cargo nextest run`. Regular testing is crucial to ensure the stability and reliability of the software.
```

**Review by axiomatic-aardvark — COMMENTED** [review]
Amazing work 🚀 just some formatting suggestions

**Comment by hopeyen** [comment]
I resolved some of the suggestions but left some personal preferences on the docs (mostly on the capitalization of phrases); personally would rather not make those commit changes, but feel free to do so when you update the docs

**Review by axiomatic-aardvark — APPROVED** [review]

**Comment by axiomatic-aardvark** [comment]
> thanks for the review. I resolved some of the suggestions but left some personal preferences on the docs (mostly on the capitalization of phrases); personally would rather not make those commit changes, but feel free to do so when you update the docs. will squash before merge

sgtm

**Pull request merged**